README.md

```diff
@@ -603,9 +603,9 @@ Here is an example of the conversion process for a pre-trained `BERT-Base Uncase
 export BERT_BASE_DIR=/path/to/bert/uncased_L-12_H-768_A-12
 
 pytorch_pretrained_bert convert_tf_checkpoint_to_pytorch \
-  $BERT_BASE_DIR/bert_model.ckpt \
-  $BERT_BASE_DIR/bert_config.json \
-  $BERT_BASE_DIR/pytorch_model.bin
+  --tf_checkpoint_path $BERT_BASE_DIR/bert_model.ckpt \
```

Comment:
Are you sure about that?
We now use a Python entry script and `sys.argv` to [pop arguments](https://github.com/huggingface/pytorch-pretrained-BERT/blob/master/pytorch_pretrained_bert/__main__.py#L16).